tests/new_test.go

```diff
@@ -18,31 +17,35 @@ type tc struct {
 }
 
 func TestNewService(t *testing.T) {
-	data := packr.New("data", "./specs")
+	folder := "./specs/"
 
 	tcs := []tc{
 		{
 			Name:    "success",
-			Spec:    data.Bytes("complete.yaml"),
+			File:    "complete.yaml",
 			WantErr: false,
 		},
 		{
 			Name:    "missing runtime",
-			Spec:    data.Bytes("missingruntime.yaml"),
+			File:    "missingruntime.yaml",
 			WantErr: true,
 		},
 		{
 			Name:    "passthrough enabled",
-			Spec:    data.Bytes("passthroughenabled.yaml"),
+			File:    "passthroughenabled.yaml",
 			WantErr: false,
 		},
 	}
 
 	for _, tt := range tcs {
 		t.Run(tt.Name, func(t *testing.T) {
-			_, err := transportd.New(
+			fileInput, err := ioutil.ReadFile(folder + tt.File)
```

Comment:
nit: You won't hit this with the way the test is set up but I'd recommend using `path.Join` to handle edge cases related to too many or too few path separators in the strings. For example, `./folder + missingruntime.yaml` would result in a test failure because of a missing slash.

Also, if you prefer to keep the content embedded in the binary instead of needing access to disk for the test then you can use the new `go:embed` directive: https://golang.org/pkg/embed/. It can expose either a string or a filesystem interface for loading files. You'd add something like
```golang
//go:embed ./specs/*
var specs embed.FS
```
and would be able to reference files like:
```golang
b, err := specs.ReadFile(tt.File)
```

Reply:
Good callout on the path join, I somehow always forget to use it. 

The embed is cool! But it took me longer than I want to admit to realize that it still wants a full relative path from this file, instead of a path relative to the folder you tell it to embed.